domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/display/DisplayGroupAdapterService.java

```diff
@@ -123,7 +123,7 @@ private WizardPageField createWizardCaseField(final DisplayGroupEntity displayGr
             .collect(Collectors.toList());
 
         return allSubTypePossibilities.stream()
-            .filter(e -> e.startsWith(displayGroupCaseFieldEntity.getCaseField().getReference()))
+            .filter(e -> e.startsWith(displayGroupCaseFieldEntity.getCaseField().getReference() + '.'))
```

Comment:
This was failing when in one event we have got claimant and claimantWorkAddress fields in Ethos definition. The overrides from claimantWorkAddress were added to claimant overrides causing front-end to show error message `Cannot find leaf for caseFieldId`